proto/sifnode/oracle/v1/types.proto

```diff
@@ -9,7 +9,7 @@ option go_package = "github.com/Sifchain/sifnode/x/oracle/types";
 // TODO: Add parameters to Genesis state ,such as minimum liquidity required to
 // create a pool
 message GenesisState {
-  repeated string address_whitelist = 1;
+  map<uint32, ValidatorWhiteList> address_whitelist = 1;
```

Comment:
So essentially this data structure is a mapping of:
`uint32 (networkid) => validatoraddress => validatorpower`
?

Reply:
yes. message ValidatorWhiteList { map<string, uint32> white_list = 1; }